pkg/context/context.go

```diff
@@ -57,7 +57,9 @@ func (cw *CensoredWriter) SetSensitiveValues(vals []string) {
 func (cw *CensoredWriter) Write(b []byte) (int, error) {
 	auditedBytes := b
 	for _, val := range cw.sensitiveValues {
-		auditedBytes = bytes.Replace(auditedBytes, []byte(val), []byte("*******"), -1)
+		if val != "" {
```

Comment:
I think this needs to be a check for all whitespace. That should cover when the output file is empty, which may have a newline in it, or just a space, etc.

Reply:
Thank you! Updated; PTAL when convenient.